src/main/java/layron/tms/model/Priority.java

```diff
@@ -1,7 +1,32 @@
 package layron.tms.model;
 
+import java.util.HashMap;
+import java.util.Map;
+
 public enum Priority {
-    LOW,
-    MEDIUM,
-    HIGH
+    LOW(1),
+    MEDIUM(2),
+    HIGH(3);
+
+    private static final Map<Integer, Priority> map = new HashMap<>();
+
+    private final int value;
+
+    Priority(int value) {
+        this.value = value;
+    }
+
+    static {
+        for (Priority priority : Priority.values()) {
+            map.put(priority.value, priority);
+        }
+    }
+
+    public static Priority valueOf(int priority) {
+        return map.get(priority);
+    }
+
+    public int getValue() {
+        return value;
+    }

```

Comment:
This code looks like it can be fully replaced with ordinal of enum

Reply:
Tried ordinal, got it wrong due to the fact that ordinal in Java starts at 0. Converter just works more predictable